logger/logger_types.go

```diff
@@ -6,8 +6,9 @@ import (
 )
 
 const (
-	timestampFormat = " 02/01/2006 15:04:05 "
-	spacer          = "|"
+	timestampFormat          = " 02/01/2006 15:04:05 "
+	spacer                   = "|"
```

Comment:
This requires whitespace on either side like the one previously at https://github.com/thrasher-corp/gocryptotrader/pull/383/files#diff-303146310b554a5a3ac1e5d3b5fc2054L50 otherwise we get output like this:

```
[INFO]| 02/01/2020 13:31:25 |OKCOIN International
```

Instead of 

```
[INFO] | 02/01/2020 13:31:25 | OKCOIN International
```

Reply:
:D another happy little accident is happening that nobody has picked up on yet with the logger 

time to fix before people notice